### ISSUE TITLE
xcat genesis base rpm not updated for 2.17

### ISSUE THREAD

**Issue by Milisha-Gupta** [xcat2/xcat-core#7479]
The xCAT Genesis base image xCAT-genesis-base-x86_64-2.14.5-snap201811190037.noarch.rpm available here (  https://xcat.org/files/xcat/repos/yum/2.17/xcat-dep/xCAT-genesis-base-x86_64-2.14.5-snap201811190037.noarch.rpm ) has not been updated for xCAT version 2.17.

Although there were relevant code changes introduced in [PR #7460] (https://github.com/xcat2/xcat-core/pull/7460) , a new RPM file has not been generated. As a result, the xCAT Genesis boot fails on newer NIC cards due to outdated drivers.

Request:
Could you please generate an updated xCAT-genesis-base RPM for both RHEL and Ubuntu to ensure compatibility with xCAT 2.17? This will help address the boot issues with newer NIC cards.

**Comment by abhishek-sa1** [comment]
@Obihoernchen Can you help here to update genesis base file?

**Comment by viniciusferrao** [comment]
@dhilst you fixed this one right?

**Comment by dhilst** [comment]
Yes, genesis-base is up to date, it ships proper kernel version now

Here is one example: 
https://xcat.org/files/xcat/repos/yum/latest/xcat-dep/rh10/x86_64/xCAT-genesis-base-x86_64-2.18.0-snap202606291947.noarch.rpm

**Comment by viniciusferrao** [comment]
> Yes, genesis-base is up to date, it ships proper kernel version now
> 
> Here is one example: https://xcat.org/files/xcat/repos/yum/latest/xcat-dep/rh10/x86_64/xCAT-genesis-base-x86_64-2.18.0-snap202606291947.noarch.rpm

Alright, so it's shipped with 2.18. Closing this one. You man want to take a look @Milisha-Gupta 

If there's something else please open another issue.
